File: Leetcode 2351 First Letter to Appear Twice.go
//2026/03/12
func repeatedCharacter(s string) byte {
    seen := make([]int, 26) 

    for _, i := range(s){
        seen[int(i - 'a')]++
        if seen[int(i - 'a')]==2{
            return byte(i)
        }
    }
    return 0
}

File: Leetcode 2351 First Letter to Appear Twice.py
# 2026/03/12
class Solution:
    def repeatedCharacter(self, s: str) -> str:
        seen = [0] * 26
        for i in s:
            ld = ord(i)-97
            if seen[ld] != 0:
                return i
            else:
                seen[ld] = 1


